docs/data/subsidy.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: http://www.neighborhoodinfodc.org/dcpreservationcatalog/
```

Comment:
@emkap01 'layout' is a specific variable used by Jekyll to tell it which format to use for the page. In this case `datasource` is the correct value.